frontend/src/components/EditTripComponents/EditAccommodationInfo.tsx

```diff
@@ -0,0 +1,44 @@
+import {Accommodation} from "../../model/Trip";
+import {ChangeEvent} from "react";
+
+type EditAccommodationInfoProps = {
+    accommodations: Accommodation[]
+    setAccommodations: (value: Accommodation[]) => void
+}
+
+export default function EditAccommodationInfo({accommodations, setAccommodations}: EditAccommodationInfoProps) {
+
+    const handleAccommodationFormChange = (event: ChangeEvent<HTMLSelectElement>, index: number) => {
+        let data: Accommodation[] = [...accommodations];
+
+        // @ts-ignore
```

Comment:
Hier die Fehlermedlung: 

TS7053: Element implicitly has an 'any' type because expression of type 'string' can't be used to index type 'Accommodation'.   No index signature with a parameter of type 'string' was found on type 'Accommodation'.

Das Ignore wurde in Zusammenarbeit mit den Coaches hinzugefügt, da diese auch keine direkte Lösung gefunden haben.